### ISSUE TITLE
Inline variable

### ISSUE THREAD

**Issue by cmstead** [cmstead/js-refactor#9]
Take a variable and inlines the value

```
function myFn () {
    var foo = 'my value';
    console.log(foo);
    return foo;
}
```

converts to

```
function myFn () {
    console.log('my value');
    returnm 'my value';
}
```

**Comment by cmstead** [comment]
Now that extract variable handles values correctly, this should be much easier to implement. This will likely be the next thing I work on.

**Comment by cmstead** [comment]
Inline variable was included in version 0.12.1.

https://marketplace.visualstudio.com/items?itemName=cmstead.jsrefactor
